source/EduCATS/Pages/Pickers/SubjectsViewModel.cs

```diff
@@ -173,7 +173,7 @@ bool setChosenSubject(SubjectModel subject)
 			return false;
 		}
 
-		void setCurrentSubjectsList(List<SubjectModel> subjects)
+		public void setCurrentSubjectsList(List<SubjectModel> subjects)
```

Comment:
Public methods should start with uppercase.
Private methods should start with lowercase.
Try to follow these rules.